app/src/androidTest/java/org/dhis2/usescases/login/LoginRobot.kt

```diff
@@ -66,7 +67,7 @@ class LoginRobot : BaseRobot() {
     }
 
     fun checkLoginButtonIsHidden() {
-        onView(withId(R.id.login)).check(matches(not(isDisplayed())))
+        onView(withId(R.id.login)).check(matches(not(isEnabled())))
```

Comment:
@Balcan the button is now always shown but no enable now? maye we can change the method name?